pkg/env/readers_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+//go:build or_dev || or_test || or_e2e
+// +build or_dev or_test or_e2e
+
+// Description: Provides configuration readers for various environments
```

Comment:
Is this even required by the linter?

If so, you should describe the file contents:
```suggestion
// Description: Unit tests for configuration readers
```